app/javascript/vue/BookPage.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <ul id="v-for-object" class="demo">
+  <li v-for="value in book">
```

Comment:
Messed up nesting. Please make sure to include `<li>` inside `<ul>`:

```html
<ul>
  <li>
  ...
...
```

Reply:
It's also recommended to always use `key` with `v-for`: https://vuejs.org/v2/guide/list.html#key 

Please provide it.